tutor/test_cc1_04_ExerciseEditingAndQA.py

```diff
@@ -157,7 +216,63 @@ def test_admin_add_the_content_analyst_role_to_a_user_7653(self):
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
-        raise NotImplementedError(inspect.currentframe().f_code.co_name)
+        self.admin.login()
+        self.admin.open_user_menu()
+        self.admin.wait.until(
+            expect.element_to_be_clickable(
+                (By.LINK_TEXT, 'Admin')
+            )
+        ).click()
+        self.admin.page.wait_for_page_load()
+        self.admin.driver.find_element(
+            By.XPATH,
+            '//a[contains(text(),"Users")]'
+        ).click()
```

Comment:
for case 7663 it logs in as content on tutor-qa.openstax.org. but then the link to edit a question for the case goes to exercises.openstax.org, which then prompts the user to re-login because it's openstax.org, not openstax-qa.org